holoviews/plotting/bokeh/element.py

```diff
@@ -104,6 +113,13 @@ class ElementPlot(BokehPlot, GenericElementPlot):
     tools = param.List(default=[], doc="""
         A list of plugin tools to use on the plot.""")
 
+    toolbar = param.ObjectSelector(default='right',
+                                   objects=["above", "below",
+                                            "left", "right", None],
+                                   doc="""
+        The toolbar location, must be one of 'above', 'below',
+        'left', 'right', None.""")
```

Comment:
It's just a convenient way to disable the toolbar altogether. I could either disable it, or warn ``tools`` or warn for ``tools`` and ``default_tools``.